src/packages/contracts/config/index.ts

```diff
@@ -193,6 +193,7 @@ export const contracts: ContractConfig[] = [
     address: {
       [ChainId.BSC_TESTNET]: '0xca11bde05977b3631167028862be2a173976ca11',
       [ChainId.BSC_MAINNET]: '0xca11bde05977b3631167028862be2a173976ca11',
+      [ChainId.SEPOLIA]: '0xca11bde05977b3631167028862be2a173976ca11',
```

Comment:
Would it make sense to add the ETHEREUM entry now, to avoid future unexpected behaviors? AFAIK it's the same address

Reply:
We have a separate ticket for this: I'd rather add the records for all the contracts at once when we get to it (the idea is that the system needs to be modular enough so that certain contracts can exist on some chains but not others, hence the partitioning).